src/amltk/optimization/trial.py

```diff
@@ -130,13 +130,53 @@ def target_function(trial: Trial) -> Trial.Report:
             options:
                 members: False
 
-    Some important properties is that they have a unique
+    Some important properties are that they have a unique
     [`.name`][amltk.optimization.Trial.name] given the optimization run,
-    a candidate [`.config`][amltk.optimization.Trial.config]' to evaluate,
+    a candidate [`.config`][amltk.optimization.Trial.config] to evaluate,
     a possible [`.seed`][amltk.optimization.Trial.seed] to use,
-    and an [`.info`][amltk.optimization.Trial.info] object which is the optimizer
+    and an [`.info`][amltk.optimization.Trial.info] object, which is the optimizer
     specific information, if required by you.
 
+    !!! tip
+    When using the `success` method, make sure to provide values for all metrics
```

Comment:
You'll need to add a blank line between these and also indent it one level. Again, try render the docs to see if it renders correctly